akka-actor/src/main/scala/akka/dispatch/affinity/AffinityPool.scala

```diff
@@ -195,7 +273,7 @@ class AffinityPoolConfigurator(config: Config, prerequisites: DispatcherPrerequi
     override val javaStrat: AffinityStrategies = AffinityStrategies.DIFFERENT_SOCKET
   }
 
-  def toStrategy(s: String): CPUAffinityStrategy = s match {
+  private def toAffinityStrategy(s: String): CPUAffinityStrategy = s match {
     case "any"              ⇒ AnyStrat
```

Comment:
Seems inconsistent to call this "AnyStrat". I'd recommend going with `<X>Strategy`